packages/angular-output-target/src/output-angular.ts

```diff
@@ -108,6 +130,18 @@ import { ProxyCmp, proxyOutputs } from './angular-component-lib/utils';\n`;
     sourceImports = cmpImports.join('\n');
   }
 
+  const proxyModules: string[] = [];
+
+  if (createAngularModules) {
+    // Generating Angular modules is only supported in the dist-custom-elements build
+    if (outputTarget.includeImportCustomElements === false) {
```

Comment:
What if `includeImportCustomElements` is elided from the config? Should we fail there as well?